requirements/dev-bookworm-requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
Shouldn't this be 3.10?

Reply:
Yes...

Reply:
OK, that just made it worse. I need to double check how we fixed this in sd-client.

Reply:
cherry-picked https://github.com/freedomofpress/securedrop-client/commit/37292714b97b1147801de408d41889b9a5d63205 to fix it!